src/odemis/gui/comp/fastem_user_settings_panel.py

```diff
@@ -0,0 +1,578 @@
+        dwell_time_overview = get_hw_config(
+            self.main_data.ebeam, self.main_data.hw_settings_config
+        ).get("dwellTime")
+        min_val, max_val, _, unit = process_setting_metadata(
+            self.main_data.ebeam, self.main_data.ebeam.dwellTime, dwell_time_overview
+        )
+        dwell_time_overview_conf = {
+            "min_val": min_val,
+            "max_val": max_val,
+            "scale": dwell_time_overview.get("scale", None),
+            "unit": unit,
+            "accuracy": dwell_time_overview.get("accuracy", 4),
+        }
+        dwell_time_acq = get_hw_config(
+            self.main_data.multibeam, self.main_data.hw_settings_config
+        ).get("dwellTime")
+        min_val, max_val, _, unit = process_setting_metadata(
+            self.main_data.multibeam, self.main_data.multibeam.dwellTime, dwell_time_acq
+        )
+        dwell_time_acq_conf = {
+            "min_val": min_val,
+            "max_val": max_val,
+            "scale": dwell_time_acq.get("scale", None),
+            "unit": unit,
+            "accuracy": dwell_time_acq.get("accuracy", 4),
+        }
+        overvoltage = self.main_data.mppc.overVoltage
+        overvoltage_conf = {
+            "min_val": overvoltage.range[0],
+            "max_val": overvoltage.range[1],
+            "unit": overvoltage.unit,
+        }
+        CONTROL_CONFIG[OVERVOLTAGE].update(overvoltage_conf)
+        CONTROL_CONFIG[DWELL_TIME_OVERVIEW_IMAGE].update(dwell_time_overview_conf)
+        CONTROL_CONFIG[DWELL_TIME_ACQUISITION].update(dwell_time_acq_conf)
+        CONTROL_CONFIG[SCINTILLATOR_HOLDER].update(
+            {"choices": list(self.main_data.samples.value.keys())}
+        )
+        if self.user_profile_data:
+            CONTROL_CONFIG[USER_PROFILE].update(
+                {"choices": list(self.user_profile_data.keys())}
+            )
+
+    def _update_selected_scintillators_layout(self):
+        current_sample = self.main_data.current_sample.value
+        if current_sample:
+            choices = []
+            labels = []
+            sample_positions = {}
+            for scintillator_num, scintillator in current_sample.scintillators.items():
+                choices.append(scintillator_num)
+                labels.append(str(scintillator_num))
+                sample_positions[scintillator_num] = scintillator.shape.position
+            layout = sample_positions_to_layout(sample_positions)
+            self.selected_scintillators.choices = choices
+            self.selected_scintillators.labels = labels
+            self.selected_scintillators.grid_layout = layout
+            self.selected_scintillators.UpdateLayout()
+        self.user_settings_panel.Layout()
+        self.user_settings_panel.Refresh()
+
+    def write_user_profile_data(self):
+        write_json(self.user_profile_config_path, self.user_profile_data)
+
+    def get_entry_control_values(self) -> Dict[str, Any]:
```

Comment:
entry does not fully make sense to me as a name, maybe `get_control_values` is already better